app/templates/buyers/brief_overview.html

```diff
@@ -211,7 +214,11 @@ <h2 class="instruction-list-item-body">
         {% endwith %}
       </ol>
     </div>
-
+    <div class="column-one-third">
+      {% if brief.publishedAt %}
```

Comment:
Yep, had that conversation with Ralph. It's not confirmed until it's publishes so we don't show it.